src/components/Icon/Icon.scss

```diff
@@ -2,11 +2,11 @@
 
 @font-face {
 	font-family: 'cloudkitIcons';
-	src:url('/src/fonts/cloudkitIcons.eot');
-	src:url('/src/fonts/cloudkitIcons.eot?#iefix') format('embedded-opentype'),
-		url('/src/fonts/cloudkitIcons.woff') format('woff'),
-		url('/src/fonts/cloudkitIcons.ttf') format('truetype'),
-		url('/src/fonts/cloudkitIcons.svg?#cloudkitIcons') format('svg');
+	src:url('/fonts/cloudkitIcons.eot');
```

Comment:
@marijanaSpajic @MatijaA will this change from src to fonts affect existing projects using fonts?